src/main/java/org/csanchez/jenkins/plugins/kubernetes/KubernetesCloud.java

```diff
@@ -425,6 +429,6 @@ public FormValidation doTestConnection(@QueryParameter URL serverUrl, @QueryPara
 
     @Override
     public String toString() {
-        return MoreObjects.toStringHelper(this).add("name", name).add("serverUrl", serverUrl).toString();
+        return "KubernetesCloud '" + name + "' serverUrl :" + serverUrl;
```

Comment:
Just didn't know where this class came from as I changed dependencies, so opted for simplest solution. I'm also not a big fan for those util library to create toString, such a simple thing to generate from IDE without being more verbose